pkg/pod/workingdir_init_test.go

```diff
@@ -27,11 +27,8 @@ import (
 
 func TestWorkingDirInit(t *testing.T) {
 	volumeMounts := []corev1.VolumeMount{{
-		Name:      "workspace",
-		MountPath: "/workspace",
```

Comment:
why did /workspace get removed? (im guessing this wasnt needed but wondering if you know why it was there and if this definitely was intentional)

Reply:
This test just tests that whatever `volumeMounts` are passed in to `workingDirInit` get appended to. In real life that's {`/workspace`, `/builder/home`, `/tekton/home`}, but for the purposes of this test it shouldn't care what the input is. In the future other volumeMounts might get added first, and it's `workingDirInit`'s job to just blindly append to them.